include/exadg/time_integration/ab_constants.h

```diff
@@ -0,0 +1,65 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_TIME_INTEGRATION_AB_CONSTANTS_H_
+#define INCLUDE_EXADG_TIME_INTEGRATION_AB_CONSTANTS_H_
+
+// C/C++
+#include <vector>
+
+// deal.II
+#include <deal.II/base/conditional_ostream.h>
+
+// ExaDG
+#include <exadg/time_integration/time_integration_constants_base.h>
+
+namespace ExaDG
+{
+class ABTimeIntegratorConstants : public TimeIntegratorConstantsBase
```

Comment:
Here, we should explain what "AB" means (similar for AM (and BDF)).

Reply:
[0e3277c](https://github.com/exadg/exadg/pull/469/commits/0e3277c19ced8ac6f66980f5bbfc840be8bdf672)